### PR TITLE
feat: generate support automatic installation-related dependencies

### PR DESCRIPTION
**Pull request by iamwangqing**
增加`fishkit/npm`，检测当前的包管理器和使用对应的包管理install
`tnf g tailwindcss`增加自动安装`tailwindcss: '^3'`依赖

---

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:

- [x] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC.
- [x] This message body should clearly illustrate what problems it solves.
- [ ] Ideally, include a test that fails without this PR but passes with it.

### Tests

- [x] Run the tests and other checks with `pnpm ci`

### Changesets

- [x] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpm changeset` and following the prompts. Changesets that add features and fix bugs should all be `patch` before we release `0.1.0`. Please prefix changeset messages with `feat:`, `fix:`, or `chore:`.

### Edits

- [x] Please ensure that 'Allow edits from maintainers' is checked. PRs without this option may be closed.

### CODE REVIEW

File: src/generate/generate_tailwindcss.ts [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import fs from 'fs-extra';
 import path from 'pathe';
+import { setConfig } from '../config/config';
 import type { Context } from '../types';
+import GeneratorHelper from './utils';
```
Comment (xiaohuoni) [inline src/generate/generate_tailwindcss.ts]: GeneratorHelper 这个工具就只有安装包的功能，不要叫这么范的名字，然后应该可以一起放在 fishkit 那边

Comment (iamwangqing) [reply src/generate/generate_tailwindcss.ts]: 好的，我改一下放到fishkit那边

**Review by xiaohuoni — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -54,6 +55,7 @@
     "compression": "^1.7.5",
     "connect-history-api-fallback": "^2.0.0",
     "cors": "^2.8.5",
+    "cross-spawn": "^7.0.5",
```
Comment (sorrycc) [inline package.json]: 用 cross-spawn 有啥充足的理由吗？没有的话就用 child_process 的 spawnSync 吧。

File: src/generate/generate_tailwindcss.ts [hunk 3]
```diff
@@ -15,6 +17,8 @@ export async function generateTailwindcss({ context }: { context: Context }) {
     }
   }
 
+  setConfig({ cwd, name: 'tailwindcss', value: true });
```
Comment (sorrycc) [inline src/generate/generate_tailwindcss.ts]: setConfig 放最后，因为 config 变更后可能让 dev server 重启，如果其他的还没好，会导致失败。

**Review by sorrycc — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (iamwangqing) [reply package.json]: 主要是考虑到一些windows上的兼容问题，用这个方便一些，这块我改一下

**Review by sorrycc — APPROVED** [review]

**Pull request merged**